File: Clojure/FizzBuzz/fizzbuzz.clj
(defn fizzbuzz [n]
  (cond (= (rem n 15) 0) "FizzBuzz"
        (= (rem n  3) 0) "Fizz"
        (= (rem n  5) 0) "Buzz"
        :else (str n)))

(defn start [n]
  (take n (map fizzbuzz (iterate inc 1))))

(println (start 10))

File: Swift/FizzBuzz/fizzbuzz.swift
func fizzbuzz(Num n: Int) {
    for i in 1...n {
        if (i % 15 == 0) {
            print("FizzBuzz")
        } else if (i % 3 == 0) {
            print("Fizz")
        } else if (i % 5 == 0) {
            print("Buzz")
        } else {
            print(i)
        }
    }
}

fizzbuzz(Num: 15)
